test/BUILD

```diff
@@ -194,3 +194,12 @@ python_test(
         "//third_party/python:six",
     ],
 )
+
+python_test(
+    name = "zip_unsafe_test",
+    srcs = ["zip_unsafe_test.py"],
+    deps = [
+        "//third_party/python:confluent-kafka",
+    ],
+    labels = ["manual"],
```

Comment:
sadly I can't just skip the test here because it fails before it even gets to the test without the updated pex tool. I'll remove this in the next PR.